### DISCUSSION TITLE
Can you show about a Win32 app which has static wrappers?

### DISCUSSION THREAD

**Discussion by japanmehta** [pywinauto/pywinauto#1055] [General]
I need help automating a win 32 application that has static wrappers and I need to automate the userID and Password fields

**Comment by vasily-v-ryabov** [comment]
Is there any specific problem accessing static texts? What `.dump_tree()` method shows?

**Reply by japanmehta** [reply]
I have a third-party .exe Windows desktop application. It has no elements I can interact with or as said it has static elements. By using pywinauto I am only being able to startup and connect to the application.I want to implement auto-populate id and password on it.

**Reply by japanmehta** [reply]
pywinauto.findwindows.ElementNotFoundError: {'title': 'eCatcher (6.6.4)', 'backend': 'win32', 'process': 17916}

Dumptree shows a couple of errors among this

**Reply by vasily-v-ryabov** [reply]
Probably you need `Application(backend="uia")`. Please read more about backends here: https://pywinauto.readthedocs.io/en/latest/getting_started.html (must read for all newcomers).
